FastSurferCNN/run_prediction.py

```diff
@@ -392,3 +395,20 @@ def handle_cuda_memory_exception(exception: RuntimeError, exit_on_out_of_memory:
         # Run model
         pred_data = eval.get_prediction(orig_fn, data_array, orig_img.header.get_zooms())
         eval.save_img(pred_name, pred_data, orig_img, seg=True)
+
+        # Run QC check (subject list may later be written out)
+        LOGGER.info("Running volume-based QC check on segmentation...")
+        seg_voxvol = np.product(orig_img.header["delta"])
+        if not check_volume(pred_data, seg_voxvol):
+            LOGGER.warning("Total segmentation volume is too small. Segmentation may be corrupted.")
+            qc_failed_subjects.append(subject)
```

Comment:
One problem with the list approach I see is, if the prediction fails on one subject for some reason, we loose the entire list of already checked cases. Might be better to have a file where this is constantly written out.